guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,465 @@
+
+```
+FROM openjdk:8-jre-alpine
+RUN apk --no-cache add --update bash coreutils
+```
+
+Once created, this image can be used as a base image by specifying the `dockerBaseImage` setting in the `build.sbt`:
+
+```
+dockerBaseImage := "<my image name>"
+```
+
+### Publishing to Docker registry - Maven
+
+We will be using the [fabric8](https://dmp.fabric8.io/) Maven plugin to containerize the application.
+
+#### Multi-module project
+
+Follow this section if your application is part of a multi-module Maven project.
+
+Add the following plugin settings to the root project `pom.xml` to register the fabric8 Maven plugin:
+
+```
+<plugin>
+    <groupId>io.fabric8</groupId>
+    <artifactId>docker-maven-plugin</artifactId>
+    <version>0.20.1</version>
+    <configuration>
+        <skip>true</skip>
+        <images>
+            <image>
+                <name>%g/%a:%l</name>
+                <build>
+                    <from>openjdk:8-jre-alpine</from>
+                    <tags>
+                        <tag>latest</tag>
+                        <tag>${project.version}</tag>
+                    </tags>
+                    <assembly>
+                        <descriptorRef>artifact-with-dependencies</descriptorRef>
+                    </assembly>
+                </build>
+            </image>
+        </images>
+    </configuration>
+</plugin>
+```
+
+When building a container image for the application, the base image [openjdk:8-jre-alpine](https://hub.docker.com/_/openjdk/) will be used to provide a working JRE 8 installation for the application to run on. The container image name expression is `%g/%a:%l` where `%g`, `%a`, and `%l` are Maven project's group id, artefact id, and the image tag. When published, our image will tagged with `latest` and `${project.version}`
+
+Add the following plugin settings on the `pom.xml` under the application's module directory:
+
+```
+<plugin>
+    <groupId>io.fabric8</groupId>
+    <artifactId>docker-maven-plugin</artifactId>
+    <configuration>
+        <skip>false</skip>
+        <images>
+            <image>
+                <build>
+                    <entryPoint>
+                        java -cp '/maven/*' -DactorSystemName=${AKKA_ACTOR_SYSTEM_NAME} -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$AKKA_REMOTING_BIND_HOST" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" com.mycompany.MainClass
+                    </entryPoint>
+                </build>
+            </image>
+        </images>
+    </configuration>
+</plugin>
+```
+
+The fabric8 Maven plugin will place all the jar files under `/maven` directory which need to be added to the JVM's classpath. Also note that we have added the Akka clustering related system properties as part of the JVM options.
+
+Replace `com.mycompany.MainClass` with the actual main class of your application.
```

Comment:
Perhaps add "usually in Akka applications this would be the class where you start your ActorSystem"?